buildroot-external/overlay/base/etc/lighttpd/conf.d/proxy_normal.conf

```diff
@@ -26,23 +26,17 @@ else $SERVER["socket"] == "[::]:443" {
   include "/etc/lighttpd/conf.d/webui.conf"
 }
 
-#-------------------------------------
-# else config auth for everything else
-#-------------------------------------
-else {
-  include "/var/etc/lighttpd_auth.conf"
```

Comment:
Here it is similar to the ssl.engine thing. I had to add back the include to every `$SERVER` definition and remove it here from the global space, because as soon as I enabled authentication, even port 80 and 443 was requesting basic authentication, which is not the wanted behaviour. Only these other ports like 8181 should get an authentication enabled, but port 80 and 443 should never be forced to an authentication.